pipeline-steps-explained.MD

```diff
@@ -0,0 +1,89 @@
+Dynatrace AppMon Pipeline steps
+-------------------------------
+Requirements: Jenkins 2.0+ with <a href="https://wiki.jenkins.io/display/JENKINS/Pipeline+Plugin">Pipeline Plugin</a>.
+
+Note: If you are using Jenkins 2.7.2 (or earlier), please ensure that your <a href="https://wiki.jenkins.io/display/JENKINS/Pipeline+Nodes+and+Processes+Plugin">Pipeline: Nodes and Processes plugin</a> version is not higher than 2.13.
+
+Note: For Maven projects, please use <a href="https://wiki.jenkins.io/display/JENKINS/Maven+Project+Plugin">Maven Integration Plugin</a> version 2.13 (or higher).
+
+Global configuration remains unchanged and is described [here](README.MD) in Global Settings section.
+
+---
+#### **appMonBuildEnvironment**
+Corresponds to 'Use Dynatrace AppMon to monitor tests' from 'Build environment' section. Wrapper that provides constant Test Run metadata for all 'appMonRegisterTestRun' steps defined.
+
+* Required parameter:
+    * `systemProfile`
+* Optional parameters:
+    * `version*` - version descriptors aggregated to "major.minor.revision.milestone" pattern
```

Comment:
you have to point all the params here: versionMajor, versionMinor, and so on; user needs to know what are their names
actually we aggregate to 'major.minor.revision **.build** milestone' in AppMon, even though the versionBuild is set automatically